internal/kafka/kafka.go

```diff
@@ -65,12 +64,13 @@ func NewConsumerEventLoop(
 	consumer *kafka.Consumer,
 	db *gorm.DB,
 ) {
-
 	sigchan := make(chan os.Signal, 1)
 	signal.Notify(sigchan, syscall.SIGINT, syscall.SIGTERM)
+	NewPayloadFieldsFromRepositoryCache := queries.NewPayloadFieldsRepository(db)
```

Comment:
This should probably just be named `payloadFieldsRepository` or something like that.  At this level, we don't know (and don't care) if its cached or not.

Let's pass the config object into the `NewPayloadFieldsRespositoy()` method here.